### PR TITLE
Steps to import production graph data on developer's graph database

### PR DESCRIPTION
**Pull request by samuzzal-choudhury**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,3 +196,29 @@ string is constructed from the following environment variables:
...
+on her/his development graph database. Following process has to be followed for the
+data import:
+
+1. SRE team dumps production data on the S3 bucket `dynamodb-migration-backups-osio`
+in `osio-dev` namespace.
+2. Use the AWS Data pipeline service to import data from graph tables.
+3. Create a pipeline with the source template `Import DynamoDB backup data from S3`.
+4. Select one of the backup folders in `s3://dynamodb-migration-backups-osio/backups/prod_edgestore/`
+as `Input S3 folder`.
+5. Provide the corresponding developer dynamodb table name as `Target DynamoDB table name`.
```
Comment (miteshvp) [inline README.md:212]: Dynamodb table should be created before this step. So two steps should talk about create dynamodb table with name and use `hk` and `rk` as index and sort key

**Review by miteshvp — CHANGES_REQUESTED** [review]
One comment.

File: README.md (hunk 1 above)
Comment (samuzzal-choudhury) [reply README.md:212]: @miteshvp this assumes that the user already has a working dev cluster instance. The steps above are all those creating a new instance.

Comment (miteshvp) [reply README.md:212]: Ok makes sense.

**Review by miteshvp — APPROVED** [review]
LGTM

**Pull request merged**